### PR TITLE
doc: add missing port

### PR DESCRIPTION
**Pull request by camilamacedo86**
Signed-off-by: Camila Macedo <camila.macedo@foundries.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ You can now run this API with:
 
 To register a device, run as root from LmP device:
 ~~~
- $ DEVICE_API=http://<IP of docker-compose host>/sign lmp-device-register [-T <TOKEN>] [-f <factory>]
+ $ DEVICE_API=http://<IP of docker-compose host>:80/sign lmp-device-register [-T <TOKEN>] [-f <factory>]
```
Comment (StealthyCoder) [inline README.md:56]: This is the default port for HTTP itself, or is there some config you want to reference here where the reference server can run? That would make sense.

Comment (camilamacedo86) [reply README.md:56]: Yes, that is the default one: https://github.com/foundriesio/factory-registration-ref/blob/main/docker-compose.yml#L17

Comment (StealthyCoder) [reply README.md:56]: I would just reference the `${API_PORT:-80}` env var there all the same. Let us say you changed the API_PORT, by accident or on purpose then this command does not reflect that.

Comment (doanac) [reply README.md:56]: i think "80" == http probably doesn't need to be explained here. Not sure the target person this would help?

Comment (camilamacedo86) [reply README.md:56]: Yep, make sense

**Pull request closed without merging**